components/library_screen/Library_Hours.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useEffect} from 'react';
+import { StyleSheet, View, Text, Linking, FlatList, ScrollView } from "react-native";
+
+import { Card, Title, Subheading, Paragraph, Button } from 'react-native-paper';
+
+import {db} from '../../firebase/config';
+import { doc, getDoc } from 'firebase/firestore';
+
+// set Locations
+const latitude = "38.34882";
+const longitude = "-97.201";
+const label = "Tabor College Library, 400 S Jefferson St, Hillsboro, KS 67063";
+
+// this detect if Phones installed Maps and open Automatically 
+const url = Platform.select({
+  ios: "maps:" + latitude + "," + longitude + "?q=" + label,
+  android: "geo:" + latitude + "," + longitude + "?q=" + label
+});
+
+const LibraryHours = () => {
+    const [libraryData, setLibraryData] = useState([]);
+
+    useEffect(() => {
+        // get Jay Shop data from Firestore
+        const docRef = doc(db, "helpful hours example", "library hours");
+        getDoc(docRef)
+            .then((doc) => {
+                const data = doc.data();
+                setLibraryData(data);
+            })
+            .catch((err) => {
+                console.log(err);
+            })
+    }, []);
+
+    // Deconstruct Just to call the open hours
+    const {"Contact information":contactInfo,"open hours": openHours } = 
+        libraryData;
+
+    return (
+        <Card style = {styles.card}>
+            <Title style ={styles.title}>{libraryData.title}</Title> 
+                <Card.Cover style = {styles.coverImage} source = {{uri: libraryData.image}}/>
+                <Paragraph style = {styles.memo}>{libraryData.memo}</Paragraph>
+                    <FlatList 
```

Comment:
Let's use the Map method instead of FlatList to map out the list

Reply:
Give me Friday to get this implemented.